packages/react-components/react-spinbutton/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-spinbutton",
-  "version": "9.0.0-beta.16",
+  "version": "9.0.0",
```

Comment:
@ling1726 please correct me if I am wrong, we don't need to change version there (#23733). It's enough to have a change file with `type: "minor"` to get a bump from `9.0.0-beta.16` to `9.0.0`. In the current state we will get a bump from `9.0.0` to `9.1.0` if I am not mistaken.

Reply:
any change file with `minor` or `patch` as the changetype will force beachball to bump to 9.0.0